cfpb/capital-framework
# Update Docs for Capital Framework v6.0.0

Updates the docsite to use the latest release from Capital Framework.

## Changes

- Updates cf-expandables to 6.0.0, adding the new modifiers to the single expandables

## Testing

- Run `npm start`
- Open http://localhost:3000/components/cf-expandables and ensure the expandables match the [design specs](https://cfpb.github.io/design-manual/page-components/expandables.html)
